src/job-exporter/src/collector.py

```diff
@@ -529,26 +529,30 @@ class ContainerCollector(Collector):
     lsof_timeout = 2 # 99th latency is 0.5s
 
     pai_services = list(map(lambda s: "k8s_" + s, [
+        # Run in master node
```

Comment:
what about nvidia device plugin, cni plugin, etc.

Reply:
https://github.com/microsoft/pai/blob/d31eb2a5e60d493cfc9c6dc50f2fa52e85e2a218/src/job-exporter/src/collector.py#L556-L558
We already collect such metrics. I can show it in grafana

Reply:
nvidia device plugin is named `nvidia-device-plugin-daemonset` instead of `nvidia-device-plugin-ctr`, there're other device plugins including amd, ib and fuse

Reply:
besides, you also missed fluentd, postgresql, internal-storage, etc.

Reply:
> nvidia device plugin is named `nvidia-device-plugin-daemonset` instead of `nvidia-device-plugin-ctr`, there're other device plugins including amd, ib and fuse

It's container name, not pod name
![image](https://user-images.githubusercontent.com/9415966/93764131-2f04ad00-fc45-11ea-8675-004a8c1bf7bb.png)

Reply:
Add postgresql, fluentd, and fuse/amd device plugin. internal storage will be removed in future, not add it

Reply:
what about ib device plugin

Reply:
fixed